webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -451,16 +451,27 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   }, [containerWidth, pinnedColumns, settings.columnWidths]);
 
   const handleCompareWidthChange = useCallback(
-    (width: number) => {
+    (newTableWidth: number) => {
+      const widthDifference = newTableWidth - comparisonViewTableWidth;
+      // Positive widthDifference: Table pane growing/compare pane shrinking
+      // Negative widthDifference: Table pane shrinking/compare pane growing
+      const newColumnWidths: Record<string, number> = { ...settings.columnWidths };
+      pinnedColumns
+        .filter(
+          (col) =>
+            col !== 'selected' && (widthDifference > 0 || newColumnWidths[col] !== minColumnWidth),
```

Comment:
**question:** is `selected` the multi-select checkbox column? Maybe make that into a constant of `MULTISELECT` or something so it stands out more